requirements/docs.txt

```diff
@@ -13,3 +13,6 @@ sphinx-copybutton>=0.3
 
 # integrations
 pytorch-lightning>=1.1
+
+# audio
+-r audio.txt
```

Comment:
I do not think it is necessary here (neither `image.txt` or `text.txt` is included)
```suggestion

# audio
-r audio.txt
```

Reply:
removed